src/main/java/Task.java

```diff
@@ -0,0 +1,38 @@
+public class Task {
+    protected String taskName;
```

Comment:
unclear variable name: does taskName refer to the description, the type of the task, or the task object itself?